### PR TITLE
building and pushing consumer wireframe in docker through the ci

### PR DESCRIPTION
**Pull request by xdrcft8000**
created and pushed a docker image to the GCP artefact registry
I added this docker image to the CI job "ci docker hurl tests", with a hurl smoke test. 
because hurl is unable to see the site after the client side js is rendered, it can only check that the page is up and that it's not displaying an error page.
Scott also added a new environment variable to ldapi to pass the RDF base url

### CODE REVIEW

**Review by rosado — DISMISSED** [review]
Please put that in a `hurl-scripts` directory, so it's consistent across this and the ldapi project. Also: rename it to `issue-318.hurl`

**Comment by rosado** [comment]
> Please put that in a `hurl-scripts` directory, so it's consistent across this and the ldapi project. Also: rename it to `issue-318.hurl`

Sorry, should have been more specific: `hurl-scripts` directory, but inside the consumer-wireframes directory.

File: docker-compose.yml [hunk 1]
```diff
@@ -6,3 +6,22 @@ services:
...
+      BASIC_AUTH_PASSWORD_SECRET_NAME: "required_but_not_used"
+      LD_API_RDF_BASE_URL: "https://dluhc-pmd5-prototype.publishmydata.com"
+    ports:
+      - "8889:80"
+    
+  frontendapp:
+    image: "europe-west2-docker.pkg.dev/swirrl-devops-infrastructure-1/public/datahost-fe-prototype:0.1.3982"
+    # platform: linux/amd64
+    environment:
+      LDAPI_BASE_URL: "http://localhost:8889"
```
Comment (RickMoynihan) [inline docker-compose.yml:25]: Rename this `LDAPI_SERVICE_URL` to avoid confusion with the concept of an RDF `BASE_URI`.

Comment (RickMoynihan) [reply docker-compose.yml:25]: Done in #326

**Pull request merged**